DGN/Models/Article.cs

```diff
@@ -23,36 +23,31 @@ public class Article
         [DataType(DataType.MultilineText)]
         public string Body { get; set; }
 
-        [DisplayName("The article image location")]
+        [DisplayName("Image name")]
```

Comment:
why?

Reply:
Because all the images will share the same path (~/images) 
The only difference is the image name

What do you think?

Reply:
then change the variable name too and define const variable for the images path

Reply:
I changed it back to location